client/src/use/useLineChart.js

```diff
@@ -4,7 +4,7 @@ export default function useLineChart({ filteredDetections, typeColorMap }) {
   const lineChartData = computed(() => {
     const _filteredDetections = filteredDetections.value;
     if (_filteredDetections.length === 0) {
-      return null;
+      return [];
```

Comment:
You changed this from a falsey to a truthy value, causing the template `v-if` to change meaning.